src/core/js/models/configModel.js

```diff
@@ -1,66 +1,51 @@
-                small: 520,
-                medium: 760,
-                large: 1024
-            },
-            _forceRouteLocking: false,
-            _canLoadData: true,
-            _disableAnimation: false
-        },
-
-        initialize: function(attrs, options) {
-            this.url = options.url;
-            // Fetch data & if successful trigger event to enable plugins to stop course files loading
-            // Then check if course files can load
-            // 'configModel:loadCourseData' event starts the core content collections and models being fetched
-            this.fetch({
-                success: function() {
-                    Adapt.trigger('offlineStorage:prepare');
-
-                    Adapt.wait.queue(function() {
-
-                        Adapt.trigger('configModel:dataLoaded');
-
-                        if (this.get('_canLoadData')) {
-                            Adapt.trigger('configModel:loadCourseData');
-                        }
-
-                        if (this.get('_defaultDirection') === 'rtl'){
-                            // We're going to use rtl style
-                            $('html').addClass('dir-rtl');
-                        }
-
-                        // Check if animations should be disabled
-                        var disableAnimationArray = this.get('_disableAnimationFor');
-                        if (disableAnimationArray && disableAnimationArray.length > 0) {
-                            for (var i = 0; i < disableAnimationArray.length; i++) {
-                                if ($("html").is(disableAnimationArray[i])) {
-                                    this.set('_disableAnimation', true);
-                                    console.log('Animation disabled.');
-                                }
-                            }
-                        }
-                    }.bind(this));
-                }.bind(this),
-                error: function() {
-                    console.log('Unable to load course/config.json');
-                }
-            });
-        },
+  var ConfigModel = Backbone.Model.extend({
+
+    defaults: {
+      screenSize: {
+        large: 900,
+        medium: 760,
+        small: 520
+      },
+      _forceRouteLocking: false,
+      _canLoadData: true,
+      _disableAnimation: false
+    },
+
+    initialize: function(attrs, options) {
+      this.url = options.url;
+      // Fetch data & if successful trigger event to enable plugins to stop course files loading
+      // Then check if course files can load
+      // 'configModel:loadCourseData' event starts the core content collections and models being fetched
+      this.fetch({
+        success: function() {
+          Adapt.trigger('offlineStorage:prepare');
+
+          Adapt.wait.queue(function() {
+
+            Adapt.trigger('configModel:dataLoaded');
+
+            if (this.get('_canLoadData')) {
+              Adapt.trigger('configModel:loadCourseData');
+            }
+
```

Comment:
did you mean to take out the code that:
* adds a `.dir-rtl` class to the HTML element
* checks to see if animations should be disabled
?

It's fine if you did mean to, but for the animation stuff isn't it work removing all other references to `_disableAnimation` at the same time?

Reply:
Yes, the language direction and animation code has been moved to app.js where it is more appropriate.

The default for `_disableAnimation` should remain on the configModel as a default is required for core.